functions.php

```diff
@@ -504,11 +506,11 @@ function get_person_markup( $person, $title=null ) {
 function display_committee_members( $people_group ) {
 	$people_group_id = $people_group->term_id;
 
-	$chair = UCF_People_PostType::append_metadata( get_field( 'people_group_chair', 'people_group_' . $people_group_id ) );
-	$vice_chair = UCF_People_PostType::append_metadata( get_field( 'people_group_vice_chair', 'people_group_' . $people_group_id ) );
-	$ex_officio = UCF_People_PostType::append_metadata( get_field( 'people_group_ex_officio', 'people_group_' . $people_group_id ) );
+	$chair = get_field( 'people_group_chair', 'people_group_' . $people_group_id );
+	$vice_chair = get_field( 'people_group_vice_chair', 'people_group_' . $people_group_id );
+	$ex_officio = get_field( 'people_group_ex_officio', 'people_group_' . $people_group_id );
 
-	$exclude = array( $chair->ID, $vice_chair->ID, $ex_officio->ID );
+	$exclude = array( $vice_chair->ID, $ex_officio->ID );
```

Comment:
Need to make sure `$vice_chair` and `$ex_officio` are set/valid post obj's before accessing their IDs here.

Reply:
I need to add $chair back in too... it was taken out for debugging purposes.